keyboards/asmodus/plaquead/info.json

```diff
@@ -0,0 +1,112 @@
+ {
+    "maintainer": "Mai The San",
+    "manufacturer": "Asmodus",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["GP3", "GP1", "GP0", "GP25", "GP11", "GP9"],
+        "cols": ["GP29", "GP27", "GP26", "GP24", "GP23", "GP22", "GP21", "GP20", "GP19", "GP18", "GP17", "GP16", "GP15", "GP14", "GP13", "GP5", "GP4"]
+    },
+    "dynamic_keymap": {
+        "layer_count": 6
+    },
+    "url": "https://paramountkeeb.com/products/plaque-a-d",
+    "layouts": {
```

Comment:
Could the following Community Layouts (/layouts/default/) be added to `layouts`:
- LAYOUT_tkl_ansi
- LAYOUT_tkl_ansi_split_bs_rshift
- LAYOUT_tkl_ansi_tsangan
- LAYOUT_tkl_ansi_tsangan_split_bs_rshift
- LAYOUT_tkl_iso
- LAYOUT_tkl_iso_split_bs_rshift
- LAYOUT_tkl_iso_tsangan
- LAYOUT_tkl_iso_tsangan_split_bs_rshift
- LAYOUT_tkl_ansi_f13
- LAYOUT_tkl_ansi_f13_split_bs_rshift
- LAYOUT_tkl_ansi_f13_tsangan
- LAYOUT_tkl_ansi_f13_tsangan_split_bs_rshift
- LAYOUT_tkl_iso_f13
- LAYOUT_tkl_iso_f13_split_bs_rshift
- LAYOUT_tkl_iso_f13_tsangan
- LAYOUT_tkl_iso_f13_tsangan_split_bs_rshift

Looking at image provided in `readme.md`, it appears the "Standard" PCB doesn't have ISO Enter key support but the "RGB" does, which means different Community Layout support per version.

Once Community Layouts have been added into `layouts` of `asmodus/plaquead/info.json`, could `"community_layouts"` element be added to `info.json` of the respective version:
### Standard
```
"community_layouts": [
    "tkl_ansi",
    "tkl_ansi_split_bs_rshift",
    "tkl_ansi_tsangan",
    "tkl_ansi_tsangan_split_bs_rshift",
    "tkl_ansi_f13",
    "tkl_ansi_f13_split_bs_rshift",
    "tkl_ansi_f13_tsangan",
    "tkl_ansi_f13_tsangan_split_bs_rshift"
],
```
### RGB
```
"community_layouts": [
    "tkl_ansi",
    "tkl_ansi_split_bs_rshift",
    "tkl_ansi_tsangan",
    "tkl_ansi_tsangan_split_bs_rshift",
    "tkl_ansi_f13",
    "tkl_ansi_f13_split_bs_rshift",
    "tkl_ansi_f13_tsangan",
    "tkl_ansi_f13_tsangan_split_bs_rshift",
    "tkl_iso_f13",
    "tkl_iso_f13_split_bs_rshift",
    "tkl_iso_f13_tsangan",
    "tkl_iso_f13_tsangan_split_bs_rshift",
    "tkl_iso",
    "tkl_iso_split_bs_rshift",
    "tkl_iso_tsangan",
    "tkl_iso_tsangan_split_bs_rshift"
],
```

Reply:
bro, just take a look closer to my picture

Reply:
[reference image](https://i.imgur.com/axShuI3.jpeg)  
Bottom PCB, of the stacked 3, has ISO Enter support whereas the middle and top PCBs do not have ISO Enter support, i.e. unable to determine which PCB is which version.